src/test/java/jenkins/plugins/git/MergeWithGitSCMExtensionTest.java

```diff
@@ -92,4 +92,4 @@ protected GitSCMExtension getExtension() {
         return new MergeWithGitSCMExtension(baseName,baseHash);
     }
 
-}
+}
```

Comment:
There should always be a newline at the end of a  file. GitHub also shows the absence of it as two red symbols. If you hover of it, it says "No newline at end of file". That's exactly what git shows on the command line when a `git show --patch <commit id>` is performed.